frontend/src/modules/machinery/components/Translation.js

```diff
@@ -53,7 +55,11 @@ export default function Translation(props: Props) {
             return;
         }
 
-        if (typeof editorContent !== 'string') {
+        // If there is no entity then it is a search term and it is
+        // added to the editor instead of replacing the editor content
```

Comment:
Übernit: could you please make sure this comment and the next one both use the same indentation level?